packages/flutter/test/material/slider_test.dart

```diff
@@ -4540,14 +4574,40 @@ void main() {
       await tester.pump();
       // changes from 1.0 -> 0.5
       expect(value, 0.5);
+      // overlay still there
+      expect(
+        material,
+        paints..circle(color: overlayColor, radius: overlayRadius, x: xPosThumb.transform(value)),
+      );
       await gesture.moveTo(endOfTheSliderTrack);
       await tester.pump();
       // changes from 0.0 -> 1.0
       expect(value, 1.0);
       expect(logs, <String>['onChangeStart', 'onChanged', 'onChanged']);
+      // overlay still there
+      expect(
+        material,
+        paints..circle(color: overlayColor, radius: overlayRadius, x: xPosThumb.transform(value)),
+      );
 
       await gesture.up();
+
+      // start release animation
       await tester.pump();
+      // go to mid-animation frame
+      await tester.pump(kRadialReactionDuration * 0.5);
+      expect(material, paints..circle(color: overlayColor, x: xPosThumb.transform(value)));
+      // verify the radius is not full
+      expect(
+        material,
+        isNot(
+          paints..circle(color: overlayColor, radius: overlayRadius, x: xPosThumb.transform(value)),
+        ),
+      );
+
+      await tester.pumpAndSettle();
+      // no overlay drawn
```

Comment:
Nit: These comments should start with a capital letter and end with a period. I know that some of them were already in this state before your PR, though.

Reply:
Should I change them all in this test then?

Reply:
If it's not too many then yes please! Otherwise, just change the ones you added, and then we can follow up with a second PR to fix the others.